packages/adapter-sveltekit/public/Hero/typescript.svelte

```diff
@@ -0,0 +1,232 @@
+<script lang="ts">
+	import { PrismicImage, PrismicRichText } from "@prismicio/svelte";
+	import { isFilled } from "@prismicio/client";
+	import type { Content } from "@prismicio/client";
+
+	export let slice: Content.PascalNameToReplaceSlice;
+</script>
+
+<section
+	data-slice-type={slice.slice_type}
+	data-slice-variation={slice.variation}
+	class="es-bounded es-fullpage-hero"
+>
+	<div
+		class={`
+        es-bounded__content
+        es-fullpage-hero__content
+        ${
+					slice.variation === "imageRight"
+						? "es-fullpage-hero__image--right"
+						: "es-fullpage-hero__image--left"
+				}
+    `}
+	>
+		{#if isFilled.image(slice.primary.image)}
+			<div>
+				<PrismicImage
+					field={slice.primary.image}
+					class="es-fullpage-hero__image"
+				/>
+			</div>
+		{/if}
+		<div class="es-fullpage-hero__content-right">
+			<div class="es-fullpage-hero__content__intro">
+				{#if isFilled.keyText(slice.primary.eyebrowHeadline)}
+					<p class="es-fullpage-hero__content__intro__eyebrow">
+						{slice.primary.eyebrowHeadline}
+					</p>
+				{/if}
+				{#if isFilled.richText(slice.primary.title)}
+					<div class="es-fullpage-hero__content__intro__headline">
+						<PrismicRichText field={slice.primary.title} />
+					</div>
+				{/if}
+				{#if isFilled.richText(slice.primary.description)}
+					<div class="es-fullpage-hero__content__intro__description">
+						<PrismicRichText field={slice.primary.description} />
+					</div>
+				{/if}
+			</div>
+		</div>
+	</div>
+
+	<style>
```

Comment:
⚠️ I see that there are some classNames not used in markup:

`.es-fullpage-hero__content-right__intro, .es-fullpage-hero__content__items, .es-fullpage-hero__item`

Right? Maybe we should quickly double-check that before copy pasting the style in all components?